.prettierrc.json

```diff
@@ -1,7 +1,7 @@
 {
     "$schema": "http://json.schemastore.org/prettierrc",
     "arrowParens": "avoid",
-    "singleQuote": false,
+    "singleQuote": true,
```

Comment:
It might be better to remove this, as `singleQuote: true` forces single quotes, but as <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52946#issuecomment-841437698> shows, double quotes are used more in this repository.
```suggestion
```